### PR TITLE
More Fastlane setup used at Acast

### PR DESCRIPTION
**Pull request by JagCesar**
I went bananas on our last hackday and created a `setup` lane. I used it yesterday when a second iOS Developer joined and it was great. Saved us so much time!

The purpose of the `setup` lane is to make sure the developer can start coding as soon as possible. Check the fastlane docs or `Fastfile` to see what is installed.

I want each developer to use their own apple-id instead of having to share a single account to iTunes Connect and Member center. Mainly because it would mean that we would have to change the password everytime somebody quits (not that this is an issue now, but I want to create something that can scale).

So in the `setup` lane I ask the user to enter the apple-id which is going to be used for developing. The apple-id is then written to a file named `appleId`. The file `appleId` is of course added to `.gitignore`.

In the `before_all` lane we check if the file `appleId` exists, if so we store the value into `ENV["DEVELOPER_EMAIL"]`. In the `Appfile` we set the property `apple_id` to `ENV["DEVELOPER_EMAIL"]`.

Maybe there's a better way of achieving this? Feedback would be great if something looks funky.

### CODE REVIEW

File: Acast/Fastfile [hunk 1]
```diff
@@ -4,19 +4,50 @@ default_platform :ios
...
+  desc "- Asks you to enter your apple-id which will be used to authorize you on iTunes Connect"
+  desc "- Copies the device frames we use for our screenshots to your home directory"
+  desc "- Installs the pods our app is dependant on"
+  desc "- Installs Xcode CLI"
+  desc "- Installs Homebrew"
+  desc "- Installs SwiftLint"
+  desc "- Installs FixCode"
+  desc "- Downloads and installs the required provisioning profiles"
+  lane :setup do
+    appleId = prompt(text: "Welcome to the Acast iOS Team!❤️\nEnter your apple-id: ")
```
Comment (KrauseFx) [inline Acast/Fastfile:27]: Should be `apple_id`

File: Acast/Fastfile [hunk 2]
```diff
@@ -4,19 +4,50 @@ default_platform :ios
 
 platform :ios do
   before_all do
+    if File.exist?("appleId")
```
Comment (KrauseFx) [inline Acast/Fastfile:7]: How about instead of doing this here, move this code to the `Appfile`, where it belongs to? :+1:

**Comment by KrauseFx** [comment]
Sorry for the late reply, I added some comments :+1:

**Comment by ohayon** [comment]
hey @JagCesar, looks like there hasn't been any movement on this PR as of recently. I am going to close it for now, but please reopen if you would like to make those changes and get it merged! thank you!!! :octopus:

**Pull request closed without merging**